### PR TITLE
Multiple line chart

### PR DESCRIPTION
**Pull request by charles161**
Added functionality to support multiple lines in the Chart Widget

### CODE REVIEW

**Comment by rrameshbtech** [comment]
Please update documentation with new expected input format for chart widget.

**Comment by charles161** [comment]
The README is edited.

File: docs/adr-04-charts-configuration.md [hunk 1]
```diff
@@ -42,6 +42,17 @@ Below is the configuration template for charts.
 }
 ```
 
+### Multiple Line Chart Configuration
+
+```
+{
+    ...
+    yAxis:"label1;label2", //multiple field names can be separated by semicolons
```
Comment (rrameshbtech) [inline docs/adr-04-charts-configuration.md]:
```suggestion
    yAxis: "label1;label2", // field names for each y-axis line in chart, separated by semicolon
```

File: docs/adr-04-charts-configuration.md [hunk 2]
```diff
@@ -42,6 +42,17 @@ Below is the configuration template for charts.
...
+    lineStroke:"#00ffff;#ff00ff",//colour for lines in the chart separated by semicolons
```
Comment (rrameshbtech) [inline docs/adr-04-charts-configuration.md]: Provide proper spacing as suggested above & provide more descriptive comment

File: src/charts/types/line-chart.component.tsx [hunk 3]
```diff
@@ -6,20 +6,34 @@ import {
...
   Tooltip,
-  ResponsiveContainer
+  ResponsiveContainer,
+  Legend
 } from "recharts";
 
 export default function(props: LineChartProperties) {
   const { data, yAxis, xAxis, lineStroke, gridStroke } = props;
 
+  function getLinesForChart() {
```
Comment (rrameshbtech) [inline src/charts/types/line-chart.component.tsx]: getChartLines?

File: src/charts/types/line-chart.component.tsx [hunk 4]
```diff
@@ -6,20 +6,34 @@ import {
...
+    return yAxis
+      .split(";")
```
Comment (rrameshbtech) [inline src/charts/types/line-chart.component.tsx]: No magic constants plz

File: src/charts/types/line-chart.component.tsx [hunk 5]
```diff
@@ -6,20 +6,34 @@ import {
...
+      .map((label, index) => (
+        <Line
+          type="monotone"
+          dataKey={label}
+          stroke={lineStroke.split(";")[index]}
```
Comment (rrameshbtech) [inline src/charts/types/line-chart.component.tsx]: split the colours constant outside the loop

File: docs/adr-04-charts-configuration.md (hunk 2 above)
Comment (rrameshbtech) [inline docs/adr-04-charts-configuration.md]: The name of the config property does not mention that it needs colour for the stroke. Please rename.

**Pull request merged**